App/Home/utils.ts

```diff
@@ -0,0 +1,180 @@
+const handleOnPress = (
+  toShow: string,
+  isOperation: boolean,
+  firstNumber: string,
+  setFirstNumber: React.Dispatch<React.SetStateAction<string>>,
+  secondNumber: string,
+  setSecondNumber: React.Dispatch<React.SetStateAction<string>>,
+  operation: string,
+  setOperation: React.Dispatch<React.SetStateAction<string>>,
+  setResult: React.Dispatch<React.SetStateAction<number>>,
+) => {
+  switch (toShow) {
+    case '=':
+      if (operation && !secondNumber) {
+        break;
+      }
+      getResult(operation, setResult, firstNumber, secondNumber);
+      cleanAfterResult(setFirstNumber, setSecondNumber, setOperation);
+      break;
+    case 'AC':
+      setFirstNumber('');
+      setSecondNumber('');
+      setOperation('');
+      setResult(0);
+      break;
+    case 'C':
+      if (secondNumber) {
+        setSecondNumber((prev: string) => prev.slice(0, -1));
+        break;
+      }
+      if (operation) {
+        setOperation('');
+        break;
+      }
+      setFirstNumber((prev: string) => prev.slice(0, -1));
+      break;
+    case '+':
+    case '/':
+    case '-':
+    case '*':
+      if (!operation) {
+        setOperation(toShow);
+      }
+      break;
+    default:
+      if (operation && !isOperation) {
+        setSecondNumber((prev: string) => prev + toShow);
+      }
+      if (!operation && !isOperation) {
+        setFirstNumber((prev: string) => prev + toShow);
+      }
+      break;
+  }
+};
+
+const getResult = (
+  operation: string,
+  setResult: React.Dispatch<React.SetStateAction<number>>,
+  firstNumber: string,
+  secondNumber: string,
+) => {
+  switch (operation) {
+    case '+':
+      setResult(parseFloat(firstNumber) + parseFloat(secondNumber));
+      break;
+    case '-':
+      setResult(parseFloat(firstNumber) - parseFloat(secondNumber));
+      break;
+    case '*':
+      setResult(parseFloat(firstNumber) * parseFloat(secondNumber));
+      break;
+    case '/':
+      setResult(parseFloat(firstNumber) / parseFloat(secondNumber));
+      break;
+    default:
+      break;
+  }
+};
+
+const cleanAfterResult = (
+  setFirstNumber: React.Dispatch<React.SetStateAction<string>>,
+  setSecondNumber: React.Dispatch<React.SetStateAction<string>>,
+  setOperation: React.Dispatch<React.SetStateAction<string>>,
+) => {
+  setFirstNumber('');
+  setSecondNumber('');
+  setOperation('');
+};
+
+const BUTTONS = [
+  {
+    isOperation: true,
+    toShow: 'C',
+    style: {
+      width: '45%',
+      paddingVertical: 10,
+      paddingHorizontal: 15,
+      backgroundColor: '#966BDE',
+    },
+  },
+  {
+    isOperation: true,
+    toShow: 'AC',
+    style: {
+      width: '45%',
+      paddingVertical: 10,
+      paddingHorizontal: 15,
+      backgroundColor: '#966BDE',
```

Comment:
Código repetido, movelo al archivo `styles.ts`.

En este caso a una variable:
```suggestion
    export const bigButtonStyle = {
      width: '45%',
      paddingVertical: 10,
      paddingHorizontal: 15,
      backgroundColor: '#966BDE',
    };
    
    Y úsalo luego:
    style: bigButtonStyle,
```